includes/classes/class-ct-db-registration.php

```diff
@@ -57,7 +57,15 @@ public function check_user_permission() {
 		 */
 		public function return_login_registration_form( $atts, $content = '' ) {
 
-			if( ! is_user_logged_in() ) {
+			$successful_registration = false;
+			if ( ! empty( $_POST ) && isset( $_POST['ctdb_user_email'] ) ) {
+				$user_email = sanitize_email( wp_unslash( $_POST['ctdb_user_email'] ) );
+				if ( $user ) {
```

Comment:
I think you accidentally removed the line which fetches the `$user` since it doesn't exist anymore and this will throw an error.